Vanila.js/JS/index.js

```diff
@@ -0,0 +1,56 @@
+
+function enterkey(){
+    let input = document.getElementById("createToDo");
+
+    if(window.event.keyCode == 13){
+        console.log(input.value);
+        if(localStorage.getItem(input.value) == null && input.value != "")
+        {    
+            localStorage.setItem(input.value,JSON.stringify(input.value));
+            draw(makecontent(input.value));
+        }
+        
+        input.value = "";
+    }
+}
+
+function makecontent(val){
+    let content = document.createElement("div");
+    let h = document.createElement("h1");
+    let txtNode = document.createTextNode(val);
+    let button = document.createElement("button");
+    
+    button.setAttribute("class","destory");
+    button.setAttribute("onclick", "remove(" +"\""+val+"\""+")")
```

Comment:
event object를 사용하는 것이 더 좋을 것 같습니다.
```suggestion
    button.setAttribute("onclick", remove)
```
:wrench:할 일 이름에 쌍따옴표(")가 들어가면 동작하지 않는 문제가 있습니다.